be/src/storage/lake/replication_txn_manager.cpp

```diff
@@ -348,6 +348,13 @@ Status ReplicationTxnManager::replicate_remote_snapshot(const TReplicateSnapshot
             src_snapshot_info.backend.host, src_snapshot_info.backend.http_port, request.src_token,
             src_snapshot_info.snapshot_path, request.src_tablet_id, request.src_schema_hash, file_converters));
 
+    for (auto& op_write : *txn_log->mutable_op_replication()->mutable_op_writes()) {
```

Comment:
these mutable_xxx will modify the txn_log if doesn't exist, whether need to check has_op_replication/has_op_writes before calling mutable_xxx?

Reply:
replication txn log must have op_replication and op_writes in it, so no need to check